react/geolocation/geolocationAutoCompleteAddress.js

```diff
@@ -131,40 +129,19 @@ export default function geolocationAutoCompleteAddress(
   return address
 }
 
-/** This function creates a map like this:
- * {
- *   "postal_code": "postalCode",
- *   "street_number": "number",
- *   "route": "street",
- *   "neighborhood": "neighborhood",
- *   "sublocality_level_1": "neighborhood",
- *   "sublocality_level_2": "neighborhood",
- *   "sublocality_level_3": "neighborhood",
- *   "sublocality_level_4": "neighborhood",
- *   "sublocality_level_5": "neighborhood",
- *   "administrative_area_level_1": "state",
- *   "administrative_area_level_2": "city",
- *   "locality": "city"
- * }
- * So it's easy to find which Google address type matches ours
- */
-function revertRuleIndex(geolocationRules) {
-  return Object.entries(geolocationRules).reduce((acc, [propName, value]) => {
-    for (let i = 0; i < value.types.length; i++) {
-      const type = value.types[i]
-
-      acc[type] = propName
-    }
+function getAddressComponent(addressComponents, types) {
+  let addressComponent
 
-    return acc
-  }, {})
-}
+  types.every((type) => {
+    addressComponent = addressComponents.find(
+      (component) =>
+        !!component.types.some((componentType) => componentType === type)
```

Comment:
I think `includes` is a bit more clear here
```suggestion
        component.types.includes(type)
```

Reply:
Yep, indeed!